examples/label.yaml

```diff
@@ -0,0 +1,9 @@
+contents:
```

Comment:
This can be excluded from the final merge; I just wanted to include what I used to generate the lockfile JSON in the PR body.